src/main/java/uk/co/ramp/covid/simulation/place/CareHome.java

```diff
@@ -9,13 +9,12 @@
 
 public class CareHome extends CommunalPlace implements Home {
 
-    private RoundRobinAllocator<Shifts> shifts;
-    private boolean isQuarantined;
+    private final RoundRobinAllocator<Shifts> shifts;
 
     public CareHome(Size s) {
         super(s);
 
-        transAdjustment = PopulationParameters.get().buildingProperties.careHomeTransmissionConstant;
+        transConstant = PopulationParameters.get().buildingProperties.careHomeTransmissionConstant;
```

Comment:
This is not refactoring, and it's not obvious why it's here.

Reply:
Looks like it arrived in one of the merge commits (4364c1). Not sure why github hasn't filtered it out (unless I had to change this here to fix the tests. Either way this is the correct variable to set, not the adjustment)